re_run.sh

```diff
@@ -152,10 +152,14 @@ then
     python src/import_other_reagent.py -f sampledata/HMS-LINCS_other_reagents.xls
     check_errs $? 'import other reagents fails'
 	
-	echo 'import test_dataset...'
-	python src/import_dataset.py -f sampledata/test_dataset.xls 
-	check_errs $? "import dataset fails"
-	
+  echo 'import test_dataset...'
```

Comment:
The indentation is all over the place in this script. This PR is not the place to fix it, but I just wanted to point it out for a future cleanup.